test/blocks/global-navigation/global-navigation.test.js

```diff
@@ -63,6 +64,54 @@ describe('global navigation', () => {
     });
   });
 
+  describe('Cross Clound Menu', () => {
+    describe('desktop', () => {
+      it('should render the Cross Clound Menu', async () => {
+        await createFullGlobalNavigation({ globalNavigation: globalNavigationCrossCloud });
+        const crossCloudMenu = document.querySelector(selectors.crossCloudMenuWrapper);
+
+        expect(crossCloudMenu).to.exist;
+        expect(isElementVisible(crossCloudMenu)).to.equal(false);
+
+        document.querySelector(`${selectors.largeMenu} ${selectors.navLink}`).click();
+
+        [...crossCloudMenu.querySelectorAll(selectors.navLink)].forEach((el) => {
```

Comment:
NodeLists already expose a `forEach` method https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach

I don't think the destructuring here is neccesary

Reply:
Good catch! I fixed it 👍